tests/FsCheck.Test/Gen.fs

```diff
@@ -119,6 +119,13 @@ module Gen =
         Gen.listOfLength length (Gen.constant v)
         |> sample1
         |> ((=) (List.init length (fun _ -> v)))
+
+    [<Property>]
+    let Shuffle (xs:Set<int>) =
+        not xs.IsEmpty ==> lazy
+        Gen.shuffle xs
+        |> sample 10
+        |> List.exists ((<>) (Set.toSeq xs))
```

Comment:
Yes, it could be, but since _shuffle_ is essentially a randomising operation, I can't think of any other property that will _always_ hold.

It's possible that a shuffle by chance will order the elements in the original order, unless, of course, you specify that a shuffle operation must always reorder at least two elements.